components/automate-cli/pkg/diagnostics/context.go

```diff
@@ -203,6 +209,73 @@ func (c *testContext) DoLBRequest(path string, opts ...lbrequest.Opts) (*http.Re
 	return nil, lastErr
 }
 
+func (c *testContext) GetVersion() (string, error) {
+	if c.version != "" {
+		return c.version, nil
+	}
+
+	httpResp, err := c.DoLBRequest("/api/v0/version")
+	if err != nil {
+		return "", err
+	}
+	defer httpResp.Body.Close() // nolint: errchech
+
+	resp := struct {
+		Build string `json:"build_timestamp"`
+	}{}
+
+	if err := json.NewDecoder(httpResp.Body).Decode(&resp); err != nil {
+		return "", err
+	}
+
+	c.version = resp.Build
+
+	return resp.Build, nil
+}
+
+func (c *testContext) IsIAMV2() (bool, error) {
+	httpResp, err := c.DoLBRequest("/apis/iam/v2beta/policy_version")
+	if err != nil {
+		return false, err
+	}
+	defer httpResp.Body.Close() // nolint: errchech
+
+	if httpResp.StatusCode == 404 {
```

Comment:
Isn't there a point in time where a version of automate will both have this endpoint and be on v1?

Reply:
Yeah, that case should be handled on L276 which returns `return vsn.Version.Major == "V2", nil` based on the endpoint response